### DISCUSSION TITLE
Ohmyzsh on Git Bash question

### DISCUSSION THREAD

**Discussion by XDeric** [ohmyzsh/ohmyzsh#9853] [Help] [answered]
I'm using git bash on windows 10, but it's very plain and I want to use ohmyzsh git command shortcuts. I'm not sure how to download it onto git bash. The curl command doesn't work, it says zsh is not installed. I've looked in the readme and I'm not sure where I can download zsh, I've tried cloning the repo but that didn't work either. 
Thank you for taking the time in looking into my problem.

**Comment by mcornella** [comment, marked as answer]
~Windows' git bash uses `msys2` under the hood~. You need to install zsh and curl via `pacman -S zsh curl`. See [the docs for installing a package](https://www.msys2.org/docs/package-management/#installing-a-package).

**Reply by XDeric** [reply]
Thanks for replying. It doesn't seem to have pacman command either.
![GitBash](https://user-images.githubusercontent.com/19171748/115901282-f7562180-a42e-11eb-8886-532f668a2844.png)

**Reply by mcornella** [reply]
I mispoke then. It looks like git bash for Windows installs mingw64 but not as part of msys. It doesn't look like there's an easy way to install packages on that: https://superuser.com/questions/701141/how-to-add-more-commands-to-git-bash. I recommend you to learn WSL instead which is a fully-functional linux environment.

**Reply by XDeric** [reply]
Thanks, looking through the superuser I got wget package onto git bash. I also installed zsh file zsh-5.8.tar.xz but not sure where it needs to go; it's on my desktop for now.
I tried the wget command
sh -c "$(wget -O- https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
but it still tells me I need to download it.

![gitbash](https://user-images.githubusercontent.com/19171748/115910686-1fe41880-a43b-11eb-8b36-ad18540cbd01.png)

**Reply by rwmitchell** [reply]
You downloaded install.sh, have you tried running: ```sh ./install.sh``` ?

(except it was written to stdout, so you're going to have to do it again and save it to a file this time)

and ```zsh``` is not ```ohymyzsh``` so you still need to get it first..

**Reply by XDeric** [reply]
oh, `sh ./install.sh ` it said no such file or directory . How do I save it to a file? Thanks

**Reply by rwmitchell** [reply]
The correct answer is to install zsh first. This command is intended to pass the contents of install.sh to sh and execute it without using an intermediary file.

**Reply by XDeric** [reply]
I installed zsh from scourceforge (zsh-5.8.tar.xz) do I need to extract from it into my git directory?

**Reply by mcornella** [reply]
You'll have an easier time using WSL or cygwin or any other Windows UNIX-like shell. [They don't even provide support for questions like that](https://github.com/git-for-windows/git/issues/2023).

**Reply by XDeric** [reply]
ok it is very tricky, Thank you guys for taking the time in helping out.
